src/main/java/com/minecolonies/core/tileentities/TileEntityWareHouse.java

```diff
@@ -91,6 +91,32 @@ public int getCountInWarehouse(@NotNull final ItemStorage storage, int count)
         return totalCount;
     }
 
+    @Override
+    public int getCountInWarehouse(@NotNull final Predicate<ItemStack> predicate, int count)
```

Comment:
why is this within the warehouse tileentity? Since it just forwards to the building we should use the building directly, also alleviating the need to load/loaded check on the TE to begin with

Reply:
could probably even use pre-existing generic building get count inventoryutils functions instead